tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/c2j_protocol_test/C2jOutputResponse.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package com.amazonaws.util.awsclientgenerator.domainmodels.c2j_protocol_test;
+
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+public class C2jOutputResponse {
+    Integer status_code;
```

Comment:
nit: use stack `int` instead of heap `Integer` since we're not making a collection